bundles/http/Citrix/2ec16a1b-9423-45a6-8fd8-b0c42c916c0f/cb5694f4-3740-45d6-9817-44d451edb2d9.js

```diff
@@ -0,0 +1,1136 @@
+                meetingArray.push(meeting.id);
+            }
+            else{
+                meetingArray.push(array[i].meetingid)
+            }
+            return {
+                "uuid":meeting.uuid,
+                "id":Number(meeting.id),
+                "host_id":meeting.host_id,
+                "topic":meeting.topic,
+                "type":parseInt(meeting.type),
+                "start_time":moment.utc(meeting.start_time).toDate(),
+                "duration":parseInt(meeting.duration),
+                "timezone":meeting.timezone,
+                "created_at":moment.utc(meeting.created_at).toDate(),
+                "join_url":meeting.join_url,
+            }
+        });
+        dataStore.save('meetings',data);
+        if(meeting.next_page_token == ""){
+            i++;
+        }
+    }while(i<array.length);
+
+    //Removing the duplicate meetingids from the array
+    let meetingSet = new Set(meetingArray);
+    meetingArray = [...meetingSet];
+}
+//Loading the meetings details from zoom SOR to cache db
+function syncMeetingsDetails(dataStore,client){
+    let dialInNumbersRecurringcounter = 0;
+    let dialIncounter = 0;
+    let dialInNumberscounter= 0;
+    let occurrenceDatacounter = 0;
+    let dialInRecurringcounter = 0;
+    for (const meetingid of meetingArray) {
+        let response = client.fetchSync(`meetings/${meetingid}`)
+        // console.log(`${meetingid} status : ${response.status}`);
+        if(!response.ok){
+            throw new Error(response.jsonSync());
+        }
+        let meeting = response.jsonSync();
+        //Setting the meetingsmap with meetingid as key and email as value to be used in Meeting Invitations API call
+        meetingsMap.set(meeting.id,{email:meeting.host_email})
+        //if meeting is a onetime(2) meeting
+        if(meeting.type == 2){
+            let onetime = {
+                "created_at":moment.utc(meeting.created_at).toDate(),
+                "duration":Number(meeting.duration),
+                "host_id":meeting.host_id,
+                "host_email":meeting.host_email,
+                "id":Number(meeting.id),
+                "join_url":meeting.join_url,
+                "password":meeting.password,
+                "settings_alternative_host":meeting.settings.alternative_hosts,
+                "start_time":moment.utc(meeting.start_time).toDate(),
+                "start_url":meeting.start_url,
+                "timezone":meeting.timezone,
+                "topic":meeting.topic,
+                "type":Number(meeting.type),
+                "recurrence_weekly_days":null,
+                "recurrence_monthly_day":null,
+                "visibility":Boolean(true)
+            }
+
+            //storing occurrences of recurring meeting 
+            let dialIn = meeting.settings.global_dial_in_countries.map(country => {
+                return {
+                    "value":country,
+                    "unique_id":(++dialIncounter).toString(),

```

Comment:
`dialIncounter` is definitely not unique.
When you run the next synchronization round, it will start from `0` again.

I suggest to use `const uuid = library.load("uuid")` library.
Unfortunately, it is possible that it is not in documentation because there is a bug. But we are working on it and when it will be resolved, you can use it here and in other places.